### ISSUE TITLE
Encoding error when enable autorize

### ISSUE THREAD

**Issue by tin-nt** [Quitten/Autorize#81]
Step to reproduce:
- Open BApp Store.
- Install Autorize.
- Enable Autorize.
 
OS:
![image](https://user-images.githubusercontent.com/43608205/152769024-53e66534-dce8-4baa-8dc4-7b45e7b787af.png)

``` Traceback (most recent call last):
  File "/Users/***/.BurpSuite/bapps/f9bbac8c4acf4aefa4d7dc92a991af2f/Autorize.py", line 7, in <module>
    from authorization.authorization import handle_message
  File "/Users/***/.BurpSuite/bapps/f9bbac8c4acf4aefa4d7dc92a991af2f/authorization/authorization.py", line 7, in <module>
    sys.setdefaultencoding('utf8')
LookupError: no codec search functions registered: can't find encoding 'utf8'

	at org.python.core.codecs$CodecState.lookup(codecs.java:1701)
	at org.python.core.codecs$CodecState.setDefaultEncoding(codecs.java:1675)
	at org.python.core.codecs.setDefaultEncoding(codecs.java:42)
	at org.python.core.PySystemState.setdefaultencoding(PySystemState.java:518)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.python.core.PyReflectedFunction.__call__(PyReflectedFunction.java:190)
	at org.python.core.PyReflectedFunction.__call__(PyReflectedFunction.java:208)
	at org.python.core.PyObject.__call__(PyObject.java:477)
	at org.python.core.PyObject.__call__(PyObject.java:481)
	at org.python.core.PyMethod.__call__(PyMethod.java:141)
	at authorization.authorization$py.f$0(/Users/tinnguyen1/.BurpSuite/bapps/f9bbac8c4acf4aefa4d7dc92a991af2f/authorization/authorization.py:340)
	at authorization.authorization$py.call_function(/Users/tinnguyen1/.BurpSuite/bapps/f9bbac8c4acf4aefa4d7dc92a991af2f/authorization/authorization.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:173)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:630)
	at org.python.core.imp.createFromSource(imp.java:583)
	at org.python.core.imp.loadFromSource(imp.java:944)
	at org.python.core.imp.find_module(imp.java:769)
	at org.python.core.PyModule.findSubModule(PyModule.java:140)
	at org.python.core.PyModule.impAttr(PyModule.java:107)
	at org.python.core.imp.import_next(imp.java:1161)
	at org.python.core.imp.import_logic(imp.java:1278)
	at org.python.core.imp.import_module_level(imp.java:1369)
	at org.python.core.imp.importName(imp.java:1528)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1285)
	at org.python.core.PyObject.__call__(PyObject.java:433)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1620)
	at org.python.core.imp.importFrom(imp.java:1595)
	at org.python.pycode._pyx4.f$0(/Users/tinnguyen1/.BurpSuite/bapps/f9bbac8c4acf4aefa4d7dc92a991af2f/Autorize.py:11)
	at org.python.pycode._pyx4.call_function(/Users/tinnguyen1/.BurpSuite/bapps/f9bbac8c4acf4aefa4d7dc92a991af2f/Autorize.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:173)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.Py.runCode(Py.java:1687)
	at org.python.core.__builtin__.execfile_flags(__builtin__.java:535)
	at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:287)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at burp.abp.<init>(Unknown Source)
	at burp.b__.a(Unknown Source)
	at burp.fhx.a(Unknown Source)
	at burp.fhx.a(Unknown Source)
	at burp.io9.lambda$installBapp$3(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by tin-nt** [comment]
Found solution, since python3 already support utf-8 encoding, and `setdefaultencoding` deprecated.
By commenting 2 lines from `authoriztion.py`:
```py
reload(sys)
sys.setdefaultencoding('utf8')
```

**Comment by tin-nt** [comment]
Found solution, since python3 already support utf-8 encoding, and `setdefaultencoding` deprecated.
By commenting 2 lines from `authoriztion.py`:
```py
reload(sys)
sys.setdefaultencoding('utf8')
```

**Comment by Quitten** [comment]
Reopened since its a bug.

**Comment by Quitten** [comment]
fixed on [8c97f9b](https://github.com/Quitten/Autorize/commit/8c97f9bf53a7a53aa5397da62df8858925b0b0e8)
